caffe2/CMakeLists.txt

```diff
@@ -808,7 +808,7 @@ if(USE_LLVM AND LLVM_FOUND)
   elseif(UNIX)
     set(LINKER_SCRIPT "${CMAKE_CURRENT_SOURCE_DIR}/version_script.lds")
     set_target_properties(torch_cpu PROPERTIES LINK_DEPENDS ${LINKER_SCRIPT})
-    set_target_properties(torch_cpu PROPERTIES LINK_FLAGS "-Wl,--version-script=${LINKER_SCRIPT}")
+    target_link_libraries(torch_cpu PRIVATE "-Wl,--version-script=${LINKER_SCRIPT}")
```

Comment:
This is infuriating, but it seems like something is going wrong in the conda build invocation of cmake that causes the -Wl,--version-script option to be dropped if it's specified as a LINK_FLAGS, but preserved if it's specified as a target_link_library (it looks like clang uses this trick to pass in an order file).  This all seems insane but if it works I'm not going to question it.